src/Microsoft.DotNet.Tools.Builder/BuilderCommandApp.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Dnx.Runtime.Common.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.ProjectModel;
+using NuGet.Frameworks;
+using System.Linq;
+
+//This class is responsible with defining the arguments for the Compile verb.
+//It knows how to interpret them and set default values
+namespace Microsoft.DotNet.Tools.Compiler
+{
+    internal class BuilderCommandApp : CompilerCommandApp
```

Comment:
Not a fan of this pattern. Once again it's inconsistent with the rest of the code in the CLI